.gitignore

```diff
@@ -96,8 +96,3 @@ fastlane/test_output
 # https://github.com/johnno1962/injectionforxcode
 
 iOSInjectionProject/
-
-# The Example App xcodeproj creates these files when a custom scheme is made
-
-.swiftpm
-Examples/MuxPlayerSwiftExample/MuxPlayerSwiftExample.xcodeproj/xcshareddata
```

Comment:
question: Do we want to be able to commit the schemes?

Reply:
Yes, we do for the default schemes.